### PR TITLE
solution with checklist

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
- [DEMO LINK](https://Zelinskyi-Serhii.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Zelinskyi-Serhii.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,64 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```
Comment (OleksiiBlezniuk) [inline src/style.css:5]: Don't use `*` selector for reseting default styles, reset it for particular elements

File: src/style.css [hunk 2]
```diff
@@ -1 +1,64 @@
...
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  list-style-type: none;
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  padding-right: 20px;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:33]: As for me - it's more logical to use margins for distance between elements

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
`is-active` modifier should work for all links
Screenshot 2023-01-26 at 16 49 46

File: src/style.css [hunk 3]
```diff
@@ -1 +1,71 @@
...
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
```
Comment (OleksiiBlezniuk) [inline src/style.css:18]: body don't have default paddings to reset

File: src/style.css [hunk 4]
```diff
@@ -1 +1,71 @@
...
+}
+
+.is-active::after {
+  margin-top: 18px;
+  content: "";
+  display: block;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:71]: It's a `костиль` - when we'll change any of sizes (font-sizes, link paddings) - your links layout will be broken

Comment (OleksiiBlezniuk) [reply src/style.css:71]: Use `position: absolute` to attach pseudoelement to bottom of it's container instead

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**